extra_docs/realtime/channels.md

```diff
@@ -0,0 +1,131 @@
+## Communication with channels
+
+Client-side applications can communicate with each other in real-time using channels which basically, allows sending and receiving messages through websocket. 
+Before using this feature, you need to create at least one channel and an appropriate policy with allowed "publish" and "subscribe" actions. Channels can be managed with [Jexia Web Management Application](https://docs.jexia.com/getting-started/user-management/).  
+
+When an application sends a message to the channel, everyone subscribed to that channel immediately receives it. If the persistent store is activated for the channel (using management application), it is also possible to get messaging history.   
+  
```

Comment:
Can we add info about maximum size of message - 64 Kb?

Reply:
Since we do not any message size restrictions **in the** SDK, I don't think it is right to add this to the SDK documentation